index.js

```diff
@@ -104,6 +134,32 @@ class PluginXrpPaychan extends PluginBtp {
       }
     }
 
+    // now uses the info protocol to make sure scales are matching
+    if (this._currencyScale !== 6) {
+      let infoResponse
+      try {
+        infoResponse = await this._call(null, {
+          type: BtpPacket.TYPE_MESSAGE,
+          requestId: await util._requestId(),
+          data: { protocolData: [{
+            protocolName: 'info',
+            contentType: BtpPacket.MIME_APPLICATION_OCTET_STREAM,
+            data: Buffer.from([ util.INFO_REQUEST_ALL ])
+          }] }
+        })
+      } catch (e) {
+        throw new Error('peer is unable to accomodate our currencyScale;' +
+          ' they are on an out of date version of this plugin. error=' +
+          e.stack)
+      }
+
+      const info = JSON.parse(infoResponse.protocolData[0].data.toString())
+      if (this._currencyScale !== 6 && info.currencyScale !== this._currencyScale) {
```

Comment:
See [comment](https://github.com/ripple/ilp-plugin-xrp-paychan/pull/31/files#r173179281)

Reply:
:+1: yeah, in this context where it gets the peer's currency scale it shouldn't care whether its own scale is the default